tutorials/physics/using_kinematic_body_2d.rst

```diff
@@ -138,6 +138,21 @@ Anything you do with ``move_and_slide()`` can also be done with ``move_and_colli
 but it might take a little more code. However, as we'll see in the examples below,
 there are cases where ``move_and_slide()`` doesn't provide the response you want.
 
+In the example above, we assign the velocity that ``move_and_slide()`` returns
+back into the ``velocity`` variable. This is because when the character collides
+with the environment, the function recalculates the speed internally to reflect
+the slowdown.
+
+For example, if your character fell on the floor, you don't want its downward
+speed to keep increasing. Instead, you want its vertical speed to be reset to
+zero.
```

Comment:
using downwards speed and vertical speed for the same thing in one sentence is confusing. I would stick with Vertical for both.